zerver/templatetags/app_filters.py

```diff
@@ -82,6 +82,9 @@ def render_markdown_path(markdown_file_path, context=None):
     if context is None:
         context = {}
 
+    if markdown_file_path.endswith('doc.md'):
+        context['integration_name'] = markdown_file_path.split('/')[0]
```

Comment:
I think this approach might have trouble with a few interesting ones like `github_webhook`.  We should probably use a field from `zerver/lib/integrations.py`, maybe something like `display_name.lower().replace(' ', '')`?